examples/cop-dem-glo-30/Copernicus_DSM_COG_10_N53_00_W115_00_DEM/Copernicus_DSM_COG_10_N53_00_W115_00_DEM.json

```diff
@@ -113,6 +124,129 @@
+      "title": "DEM QuickLook Absolute",
+      "description": "Quicklook of Digital Elevation Model wiuth shaded relief representation (absolute)",
+      "role": [
+        "overview"
+      ]
+    },
+    "edm_ql": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_EDM_QL.tif",
+      "type": "image/tiff",
+      "title": "Editing mask QuickLook",
+      "description": "QuickLook of Editing Mask",
+      "role": [
+        "overview"
+      ]
+    },
+    "flm_ql": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_FLM_QL.tif",
+      "type": "image/tiff",
+      "title": "Filling mask QuickLook",
+      "description": "QuickLook of Filling Mask",
+      "role": [
+        "overview"
+      ]
+    },
+    "wbm_ql": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_WBM_QL.tif",
+      "type": "image/tiff",
+      "title": "Water Body mask QuickLook",
+      "description": "QuickLook of Water Body Mask",
+      "role": [
+        "overview"
+      ]
+    },
+    "hem_ql": {
+      "href": "../../tests/data-files/external/PREVIEW/Copernicus_DSM_COG_10_N53_00_W115_00_HEM_QL.tif",
+      "type": "image/tiff",
+      "title": "Height Error mask QuickLook",
+      "description": "QuickLook of Height Error Mask",
+      "role": [
+        "overview"
+      ]
+    },
+    "edm": {
+      "href": "../../tests/data-files/external/AUXFILES/Copernicus_DSM_COG_10_N53_00_W115_00_EDM.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "Editing mask",
+      "description": "Editing Mask",
+      "role": [
+        "data",
+        "data-mask"
+      ]
+    },
+    "flm": {
+      "href": "../../tests/data-files/external/AUXFILES/Copernicus_DSM_COG_10_N53_00_W115_00_FLM.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "Filling mask",
+      "description": "Filling Mask",
+      "role": [
+        "data",
+        "data-mask"
+      ]
+    },
+    "wbm": {
+      "href": "../../tests/data-files/external/AUXFILES/Copernicus_DSM_COG_10_N53_00_W115_00_WBM.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "Water Body mask",
+      "description": "Water Body Mask",
+      "role": [
+        "data",
+        "data-mask"
+      ]
+    },
+    "hem": {
+      "href": "../../tests/data-files/external/AUXFILES/Copernicus_DSM_COG_10_N53_00_W115_00_HEM.tif",
+      "type": "image/tiff; application=geotiff; profile=cloud-optimized",
+      "title": "Height Error mask",
+      "description": "Height Error Mask",
+      "role": [
+        "data",
+        "data-mask"
+      ]
+    },
+    "acm": {
+      "href": "../../tests/data-files/external/AUXFILES/Copernicus_DSM_COG_10_N53_00_W115_00_ACM.kml",
+      "type": "application/vnd.google-earth.kml+xml",
+      "title": "Height Error mask",
```

Comment:
I feel like the title should be all capitalized words, and the description should only have the first one capitalized (and maybe a bit more text after it).